modules/nw-sriov-configuring-operator.adoc

```diff
@@ -1,46 +1,36 @@
 // Module included in the following assemblies:
 //
-// * networking/multiple_networks/configuring-sr-iov.adoc
+// * networking/hardware_networks/configuring-sriov-operator.adoc
 
-[id="configuring-sr-iov-operator_{context}"]
+[id="nw-sriov-configuring-operator_{context}"]
 = Configuring the SR-IOV Network Operator
 
-The SR-IOV Network Operator adds the `SriovOperatorConfig.sriovnetwork.openshift.io` Custom Resource Definition (CRD) to {product-title}.
-You can configure the SR-IOV software compoenents by modifying the default 
-SriovOperatorConfig Custom Resource (CR).
+The SR-IOV Network Operator adds the `SriovOperatorConfig.sriovnetwork.openshift.io` CustomResourceDefinition resource to {product-title}.
```

Comment:
@bergerhoffer still trying to finalize a coherent proposal for these. It presents itself as intractable.

So like our other objects, like ReplicaSet and so forth, a custom resource definition is defined by creating a CustomResourceDefinition object. (In various places on the Internet, both seem to be abbreviated as CRD, because no consistent style exists for any of this.)

I might have another proposal attempt for these soon, maybe.

This was an attempt to use `<meta.name> <ResourceName>` as an approach. So the `SriovOperatorConfig.sriovnetwork.openshift.io` CustomResourceDefinition object is how SriovOperatorConfig as a resource comes to be.

_Custom Resource Definition (CRD)_ is an artifact of my not having understood that that construction simply originated out of someone abbreviating CustomResourceDefinition as CRD and my thinking it was A Big Proper Noun Thing that was abbreviated. It's not; Developers just like to abbreviate it as CRD, _because_.

All of this is just misunderstanding, all around, I think. (CR is a little different; Creating a CustomResourceDefinition does indeed provide a new OCP custom resource, which we're abbreviating as CR.)